src/utils/breadcrumbItems.ts

```diff
@@ -0,0 +1,53 @@
+import { remult } from "remult";
+import { Founder } from "@/entities/Founder";
+import { User } from "@/entities/User";
+import { texts } from "./texts";
+import { api } from "@/app/api/[...remult]/route";
+
+export type BreadcrumbItem = {
+  href: string;
+  title: string;
+};
+
+const foundersRepo = remult.repo(Founder);
+const usersRepo = remult.repo(User);
+
+// FOUNDERS
+export const foundersBreadcrumb = {
+  href: "/founders",
+  title: texts.founders,
+}
+
+export const founderDetailBreadcrumb = async (founderId: string) => {

```

Comment:
A tady bych přidal návratový typ:
`export const founderDetailBreadcrumb = async (founderId: string): Promise<BreadcrumbItem> => {`

A stejně tak u zbytku funkcí a itemů.

Reply:
udělal jsem to tak, akorát jsem na to vytvořil typ, který jsem použil u všech.